toolkit/tools/precacher/precacher.go

```diff
@@ -258,7 +290,34 @@ func getPackageRepoPaths(repoUrl string) (packages []string, err error) {
 	return
 }
 
-// downloadMissingPackages will attemp to download each package listed in rpmSnapshot that is not already present in the
+// getPackageRepoUrlsFromRepoFile returns a list of packages available in the given repoUrl by running repoquery
+func getPackageRepoUrlsFromRepoFile() (packageURLs []string, err error) {
```

Comment:
Nit-pick:
```suggestion
// getPackageRepoUrlsFromRepoFiles returns a list of packages available in all RPM repos listed in the system's .repo files.
func getPackageRepoUrlsFromRepoFiles() (packageURLs []string, err error) {
```

Reply:
Ah, hold over from when it did them one at a time.